### DISCUSSION TITLE
Is Anyone Else Experiencing Random Crashes?

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenLoco/OpenLoco#1971] [General]
At this point, the original disc version of the game is more playable on Windows 10 than OpenLoco. My game keeps crashing for no apparent reason, and I don't seem to be alone, judging by the number of crash report issue submissions. Does anyone have some sort of solution, even if its temporary, that can remedy this? It's hard to enjoy a game that can crash at completely random intervals. Also, who else is experiencing this issue?

**Comment by AaronVanGeffen** [comment]
When did this start? It sounds like it could be related to https://github.com/OpenLoco/OpenLoco/issues/1968, which would have been introduced in v23.05.

Assuming you're playing on the v23.05, could you try v23.04.1 to see if that makes a difference?

**Reply by Derpercoaster** [reply]
I have yet to see the title screen crash. I think it's more related to #1653. That said, it seems to happen for me at random times in the month, not just at the end. If I load an autosave from before the crash, it will predictably crash at the same exact time.

**Reply by AaronVanGeffen** [reply]
The bug I referred to is bigger than the title screen as well. It seems to occur when there's tree interaction, including AI.

Could you try if the problem also occurs on v23.04.1?

**Reply by duncanspumpkin** [reply]
@Derpercoaster can you try 23.06.1 it shouldn't have any of the crashes.

**Reply by Derpercoaster** [reply]
23.06.1 crashes almost immediately on startup for me now. I sometimes only get 30 seconds of gameplay before it crashes.

**Reply by AaronVanGeffen** [reply]
Could you try turning off display scaling? We recently fixed a bug to do with that, after v23.06.1 was released.

**Reply by Derpercoaster** [reply]
That might fix it. I just decided to play on 23.06.0 instead, but I can check.

**Comment by FenderBender2** [comment]
I'm getting constant crashes on 23.10. Is this still being investigated or should I upload a couple of dump files?

**Reply by AaronVanGeffen** [reply]
We're not aware of anything in particular, so crash dumps would help us investigate, yes.
